contracts/source/IUniswapV2Pair.sol

```diff
@@ -7,5 +7,5 @@ interface IUniswapV2Pair {
     function price0CumulativeLast() external view returns (uint256);
     function price1CumulativeLast() external view returns (uint256);
 
-	function getReserves() external view returns (uint112 _reserve0, uint112 _reserve1, uint32 _blockTimestampLast);
+    function getReserves() external view returns (uint112 _reserve0, uint112 _reserve1, uint32 _blockTimestampLast);
```

Comment:
All of the other files use tab to indent.  The proper fix here would be to fix this file to be consistent with the rest of the repository.  Not your fault though, so I'll accept this change as-is since it at least makes this file internally consistent.